ui/src/app/views/workspace-about.tsx

```diff
@@ -0,0 +1,118 @@
+import {WorkspacePermissions} from 'app/utils/workspace-permissions';
+import {ResearchPurposeDescription, ResearchPurposeItems, specificPopulations} from './workspace-edit';
+
+const styles = reactStyles({
+  mainHeader: {
+    fontSize: '20px', fontWeight: 600, color: colors.purple[0], marginBottom: '0.5rem',
+    display: 'flex', flexDirection: 'row'
+  },
+  sectionHeader: {
+    fontSize: '16px', fontWeight: 600, color: colors.purple[0], marginTop: '1rem'
+  },
+  sectionText: {
+    fontSize: '14px', lineHeight: '24px', color: colors.purple[0], marginTop: '0.3rem'
+  }
+});
+
+
+export const WorkspaceAbout = withCurrentWorkspace()(
+  class extends React.Component<{workspace: WorkspaceData},
+    {workspacePermissions: WorkspacePermissions}> {
+    constructor(props) {
+      super(props);
+      this.state = {
+        workspacePermissions: new WorkspacePermissions(props.workspace)
+      };
+    }
+
+    getSelectedResearchPurposeItems() {
+      return ResearchPurposeItems.filter((item) =>
+        this.props.workspace.researchPurpose[item.shortName]).map((item) => {
+          let content = item.shortDescription;
+          if (item.shortName === 'otherPurpose') {
+            content += ': ' + this.props.workspace.researchPurpose.otherPurposeDetails;
+          }
+          if (item.shortName === 'diseaseFocusedResearch') {
+            content += ': ' + this.props.workspace.researchPurpose.diseaseOfFocus;
+          }
+          return content;
+        });
+    }
+
+    getSelectedPopulations() {
+      return specificPopulations.filter(sp =>
+        this.props.workspace.researchPurpose.populationDetails.includes(sp.object))
+        .map(sp => sp.ubrLabel);
+    }
+
+    getSelectedPopulationsSlice(left: boolean) {
+      const populations = this.getSelectedPopulations();
+      const populationsHalfLen = sliceByHalfLength(populations);
+      if (left) {
+        return populations.slice(0, populationsHalfLen);
+      } else {
+        return populations.slice(populationsHalfLen);
+      }
+    }
+
+    render() {
+      const workspace = this.props.workspace;
+      const {workspacePermissions} = this.state;
+      const selectedResearchPurposeItems = this.getSelectedResearchPurposeItems();
+      const rpItemsHalfLen = sliceByHalfLength(selectedResearchPurposeItems);
+      return <FadeBox style={{margin: 'auto', marginTop: '1rem', width: '98%'}}>
+        <div style={styles.mainHeader}>Research Purpose
+          <Clickable disabled={!workspacePermissions.canWrite}
+                     data-test-id='edit-workspace'
+                     onClick={() => navigate(
+                       ['workspaces',  workspace.namespace, workspace.id, 'edit'])}>
+            <EditComponentReact disabled={!workspacePermissions.canWrite}
+                                style={{marginTop: '0.1rem'}}/>
+          </Clickable>
+        </div>
+        <div style={styles.sectionText}>{ResearchPurposeDescription}</div>
```

Comment:
I know Lou's mocks included this, but it feels kind of awkward to have such a long piece of boilerplate text being shown to all users who ever visit any workspace. Maybe it's worth quickly checking w/ Karthik or Lou on whether this is really the intent?

If we do need to keep this in, I'd argue it should be towards the bottom, so more useful content (e.g. the workspace description) is above the fold.

Reply:
I totally agree :/ will check with karthik